CHANGELOG.md

```diff
@@ -14,7 +14,12 @@ Released: TBD
 
 ### Minor Changes
 
-- None
+- [TBD] Use commander's new `.conflicts()` to check for mutually-exclusive CLI
```

Comment:
Tip: I usually check the last PR/issue number on GitHub (there is continuous numbering) and set number in advance :). If do that just before push there is a high chance that number will be correct